internals/helpers/package.js

```diff
@@ -71,24 +35,14 @@ function addDependency(dependency) {
  */
 
 function updatePackageJsonDependencies(dependency) {
```

Comment:
Can this be changed to

```
async function updatePackageJsonDependencies(dependency) {
  package_config.addDependency([dependency, "latest"]);
}
```

not sure why we used a Promise in the first place.

Reply:
Actually, I don't even know why this needs to be async. is `package_config.addDependency` async? We should also decide if we're going to do async everywhere. I did an async lexer because it let's me chain steps easily.

Reply:
I think that function is deprecated anyway. Will leave for now, as we will refactor modules into separate packages anyway. Suggestion fixed in 35042aac97556daefd8dbcac71fb66f6c480a289